be1-go/cli/mod.go

```diff
@@ -47,34 +47,48 @@ func Serve(cliCtx *cli.Context, user string) error {
 	otherWitness := cliCtx.StringSlice("other-witness")
 
 	pk := cliCtx.String("public-key")
-	if pk == "" {
-		return xerrors.Errorf("%s's public key is required", user)
-	}
 
-	// decode public key and unmarshal public key
-	pkBuf, err := base64.URLEncoding.DecodeString(pk)
-	if err != nil {
-		return xerrors.Errorf("failed to base64url decode public key: %v", err)
-	}
+	// compute the client server address
+	clientServerAddress := fmt.Sprintf("%s:%d", publicAddress, clientPort)
 
 	point := crypto.Suite.Point()
 
-	err = point.UnmarshalBinary(pkBuf)
-	if err != nil {
-		return xerrors.Errorf("failed to unmarshal public key: %v", err)
-	}
-
 	// get the HubType from the user
 	var hubType = hub.HubType(user)
 
-	// compute the client server address
-	clientServerAddress := fmt.Sprintf("%s:%d", publicAddress, clientPort)
+	// TODO
+	var h *standard_hub.Hub
+	var err error
 
-	// create user hub
-	h, err := standard_hub.NewHub(point, clientServerAddress, log.With().Str("role", user).Logger(),
-		lao.NewChannel, hubType)
-	if err != nil {
-		return xerrors.Errorf("failed create the %s hub: %v", user, err)
+	if pk == "" {
```

Comment:
Maybe you should also add a `Value: "",` to the publicKeyFlag in pop.go so that we don't have to specify --pk="" if we don't want to check the public key

Reply:
After having checked there is no need to do that to have the behavior you've mentioned.
Currently if you specify any --pk it will require to be correct
But if you omit it, it will run it without it